doc/rfcs/004-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -145,8 +145,7 @@ People will collaborate less often. Given the additional steps needed to start c
 
 ##### What unresolved questions do you expect to resolve through the RFC process before this gets merged?
 
-- If I invite a trusted collaborator to join my portal, and that person has multiple Atom windows open, can we show the invitation only in the frontmost window?
-- If I invite a trusted collaborator to join my portal, and that person has Atom windows open on multiple computers, are we OK with showing the invitation on each computer?
+- If I invite a trusted collaborator to join my portal, and that person has multiple Atom windows open, can we show the invitation only in the frontmost window? Similarly, if that person has Atom windows open on multiple computers, are we OK with showing the invitation on each computer? _Answer_: The invitation will appear in all open Atom windows where the user is signed into Teletype. Once the user accepts or declines the invitation on in one Atom window, Teletype will dismiss the invitation in the other Atom windows. [[discussion](https://github.com/atom/teletype/pull/344#pullrequestreview-105123633)]
```

Comment:
I think that's a good solution. It also helps in another scenario: When you wanna "snooze" an invitation. If you get invited and want to join, but still need to spend a few minutes finishing up something, then the invitation dialog blocks your editor. So in this case, you could just open a new window (where Teletype shows the invitation too), then click on the X of the current window. This will only close the dialog, but keep the invitation active (in the other window).